src/eliottree/_cli.py

```diff
@@ -285,6 +286,11 @@ def main():
                         dest='print_current_config',
                         action='store_true',
                         help='''Show the current effective configuration.''')
+    parser.add_argument('--chunk-size',
```

Comment:
Please add a `metavar` key, I would suggest calling it `LINES` so that it's clear what the unit of size is.